---
layout: default
---

<div class="container">


<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Openings</b></h2>
	<ul>
		<li> PhD Scholarships ($39,100 tuition fee plus $25,861 stipend contribution for the duration of PhD) and honours projects are immediately available. Successful candidates will undertake 
		exciting new projects in the general areas of Systems and Computational Biology.</li>
		<li> Summer Research Scholarships ($513 per week for 8 weeks) are available for third year undergraduates who are interested
		in exploring either 'wet' or 'dry' research projects in computational systems biology and consider pursuing an honours project in our group.</li>
	</ul>
	
	For postgraduate applicants, all 'wet' projects require experience with cell culturing and good understanding of basic experimental protocols (e.g. qPCR, westernblotting) and preferably experience of (single cell-)sequencing and
	mass spectrometry-based proteomics. All 'dry' projects require programming skill
	in at least one programming/scripting language (R, Perl, Python, Java, C++, C and Matlab). There are ample opportunity to
	work closely with biologists, statisticians, bioinformaticians, and computer scientists.
	<br><br>
	To Apply: Please send CV via email to Dr. Pengyi Yang (<a href="mailto:pengyi.yang@sydney.edu.au">pengyi.yang [at] sydney.edu.au</a>)
	<br><br>
	More information about scholarship opportunity at the University of Sydney can be found <a href="https://sydney.edu.au/study/study-options/postgraduate-research.html">here</a>.<br>
	For obtaining scholarships and top-ups from CMRI, please read <a href="https://www.cmri.org.au/Research/For-Students">here</a>.<br><br>
	
	Below is an <b>incomplete</b> list of research themes and potential/ongoing projects in our lab. Candidates are welcome to get in touch and discuss these 
	and/or any other potential projects aligned with one or more of our research themes.<br><br>
	
	</div>
</div>




<h2 class="projects"><b>Theme: Mapping and modelling trans-regulatory networks</b></h2>

<div class="section row">
    <div class="image col-md-6">
        <img class="img-responsive" alt="Project 1 illustration" src="/images/projects/TransReg.png"/ Align="Left">
    </div>
    <div class="col-md-7">
        <h3 class="heading">Project 1: Reconstructing trans-regulatory networks by trans-omic data integration</h3>
        <div class="content">
            A major initiative in our group is to integrate trans-omics datasets generated by state‐of‐the‐art mass spectrometer (MS) and next 
			generation sequencer (NGS) from various cell systems.
            The goal is to reconstruct trans-regulatory networks and to understand how different regulatory machineries (e.g. signalling, transcription, and epigenomics) 
            co-operate to define cell states, functions, and fates.<br><br>
            
            We have previously developed various computational methods to integrate the multi-layered trans‐omics datasets generated during naive to formative pluripotency transition 
            in ESCs (<a href="https://doi.org/10.1016/j.cels.2019.03.012">Yang et al. 2019</a>). Our current research project 
			aims to further this study by developing methods to characterise signaling cascades, transcriptional networks, and translational protein networks and their cross‐talks. 
			This project will provide a unique opportunity for developing computational methods for comprehensive understanding of cell systems and their decision-making process.
        </div>
    </div> 
    
    <div class="col-md-12">
       <h3 class="heading">Project 2: Understanding the formation of fat tissue</h3>
       <div class="content">
            The understanding of fat tissue formation is critical for treating various metabolic and obesity related diseases.
            In collaboration with Prof. David James lab, we have developed several machine learning approaches to elucidate the
            signaling cascades in fat cells (<a href="https://doi.org/10.1093/bioinformatics/btv550">Yang et al. 2016</a>; <a href="https://doi.org/10.1371/journal.pcbi.1004403">Yang et al. 2015</a>;
			<a href="https://dx.doi.org/10.1016%2Fj.cmet.2013.04.010">Humphrey et al. 2013</a>). We have now also profiled the 
            fat cell (3T3-L1) differentiation at matched time points on multiple omic levels to understand the trans-regulatory networks underlying this process. 
            You will get involved in developing key statistical models to
            integrate trans-omics data and infer their feedback regulation during adipogenesis, the maturation process of fat cells.
      </div>
   </div>     

   <div class="col-md-12">     
      <h3 class="heading">Project 3: Stem/progenitor cell differentiation and iPSC reprogramming</h3>
      <div class="content">
        The ability of stem/progenitor to self-renew and differentiate into specialised cell types in the body makes them the key cellular tools for stem cell-based therapy, 
        whereby any specific cell populations, tissues, and organs compromised by disease, injury or ageing can be replaced and replenished indefinitely. 
        While tremendous progress has been made in stem cell therapy, attempts to direct stem/progenitor cells to clinically useful cell types have often been fraught with 
        imprecise and incomplete differentiation, a common cause of tumorigenesis, owing to a lack of comprehensive understanding of trans-regulatory networks 
        that underline cell identities and cell-fate decisions.<br><br>
        
        Leveraging on the recent advances in mass spectrometry (MS) and next-generation sequencing (NGS), this project aims for experimentally profiling the trans-regulatory networks stem/progenitor
        using trans-omics. Complementary to this, we will also profile the trans-regulatory networks during somatic cell reprogramming to induced pluripotent stem cells (iPSCs). 
        Generation of these data will allow us to model and subsequently modulate trans-regulatory networks for highly precise stem/progenitor differentiation for use in stem cell-based therapy.
      </div>
   </div>
</div>
<hr>


<h2 class="projects"><b>Theme: Single-cell biology and omics</b></h2>

<div class="section row">
    <div class="image col-md-10">
        <img class="img-responsive" alt="Project 2 illustration" src="/images/projects/scMerge.png"/>
    </div>
    <div class="col-md-7">
        <h3 class="heading">Project 1: Multiple single-cell RNA-seq data integration</h3>
        <div class="content">
            We are working on multiple methodological development as well as lab experiment projects on using single-cell omic technologies to understand complex biological systems.
            One key research direction is the integration of multiple single-cell RNA-seq datasets. To this end, we have developed a computational method together with Prof. Jean Yang's
            group for multiple single-cell RNA-seq data integration (<a href="https://doi.org/10.1073/pnas.1820006116">Lin et al. 2019</a>).<br><br>
            
            We are now extending this method for ultra-large scale scRNA-seq data integration and for using diverse gene characteristics. Working on this project, you will be exposed to 
            various state-of-the-art statistical methods and their application to the latest biotechnology of single-cell omics.
        </div>
    </div>
    
    <div class="col-md-12">
        <h3 class="heading">Project 2: Cell type identification using statistical learning</h3>
        <div class="content">
			Cell type identification is a key step in scRNA-seq experiments. We are in process of developing better unsupervised, semi-supervised, and fully supervised learning methods
			for cell type identification from multiple scRNA-seq datasets.<br><br>
			
			One interesting finding so far is that not all 'similarity' metrics are equally useful in analysing scRNA-seq data (<a href="https://doi.org/10.1093/bib/bby076">Kim et al. 2018</a>).
			Note that most of the clustering and classification methods utilise some kind of similarity metric to compare cells. Therefore, the finding has great implication on both the development
			of computational methods for scNRA-seq data analysis and the biology of individual cells. This project is computational and requires a good grasp of machine learning techniques. Candidates also have
			opportunity to explore deep learning methods and their application in this domain.
        </div>
    </div>
    
    <div class="col-md-12">
        <h3 class="heading">Project 3: Single-cell profiling of ESC differentiation to multiple lineages</h3>
        <div class="content">
			This project involves experimental profiling of single cells in ESC population and during their differentiation to multiple cell lineages. The 10x genomics platform at CMRI is
			ideal for this project and accessible to our lab. Together with Prof. <a href="https://www.cmri.org.au/Research/Research-Units/Embryology/Our-People">Patrick Tam's lab</a>, 
			we have already established single-cell sequencing and epigenomic sequencing (e.g. ATAC-seq) platforms at CMRI.<br><br>
			
			The candidate will be uniquely positioned to understand ESC differentiation and early embryogenesis. Such knowledge will be instrumental in developing stem cell-based therapy.			
        </div>
    </div>
    
</div>
<hr>

<h2 class="projects"><b>Theme: Computational methodology innovation in bioinformatics</b></h2>

<div class="section row">
	<div class="col-md-12">
        <h3 class="heading">Project 1: Machine learning application in trans-omics</h3>
        <div class="content">
			Build on top of mapping and modelling trans-omic networks, a PhD position is available in developing and applying machine 
			learning models to multi-layered trans‐omics datasets generated by MS-based phospho(proteomics) and NGS-based sequencing.<br><br>

			Our research project, funded by the Australian Research Council (ARC), aims to develop novel machine learning algorithms 
			to analyse and integrate large‐scale MS‐based omics data with ultra‐fast NGS‐based omics data generated from complex 
			biological systems. Characterising the signaling cascades, transcriptional networks, and translational protein networks 
			and their cross‐talks are critical for comprehensive understanding of complex biological systems. Our large‐scale 
			multi-layered trans‐omics data generated from state‐of‐the‐art technological platforms provide a unique opportunity to 
			uncover novel biology and molecular mechanisms that are critical for treating complex diseases and personalised medicine. 
    	</div>
    </div>
	<hr>

    <div class="col-md-12">
        <h3 class="heading">Project 2: Application of deep learning in bioinformatics</h3>
        <div class="content">
			Deep learning is the latest development in machine learning that has been successfully utilised to address 
			many bioinformatics applications. We are interested in its application such as in transcription factor (TF)
			binding site identification and target gene prediction.<br><br>
			
			The advance in chromatin immunoprecipitation followed by ultrafast sequencing (ChIP-seq) allows the profiling of TF binding sites 
			genome-wide in a population of cells. The massive amount of sequencing data generated from these genome-wide profiling of TF 
			requires sophisticated computational algorithms to be developed for accurately identifying TF binding sites.
			In this project, we aim to develop and apply deep learning models for 
			predicting TF binding sites by integrating ChIP-seq data with other biological knowledge. 
			This project aims to develop and apply cutting-edge deep learning algorithms for solving a key biological problem.
			You will get involved in all aspects of the development including algorithm design, implementation and testing.
        </div>
    </div>
    
    <div class="image col-md-5">
        <img class="img-responsive" alt="Project 4 illustration" src="/images/projects/AdaSampling.png"/>
    </div>
    <div class="col-md-7">
        <h3 class="heading">Project 3: Application of semi-supervised learning in bioinformatics</h3>
        <div class="content">
        
        Identification of transcription factor target genes using machine learning
        
            We have recently developed several semi-supervised and/or knowledge-based learning methods for learning with only positive examples and in the presence of data noise 
            (e.g. <a href="https://doi.org/10.1109/TCYB.2018.2816984">Yang et al. 2018</a>). 
            We are now extending semi-supervised learning methods for various bioinformatics application. This includes the identification of transcription factor target
            genes in differentiating mouse embryonic stem cells (mESCs) by integrating transcriptomics, proteomics, and epigenomics data; the classification of cell types from single-cell
            RNA-seq data.<br><br>
            
            Candidates involved in this project we learn state-of-the-art machine learning methods and their applications to cutting-edge systems biology.
        </div>
    </div>
	
    <div class="col-md-12">
        <h3 class="heading">Project 4: Pathway reconstruction and annotation using statistical models and trans-omics data sets</h3>
        <div class="content">		
			While all cells from a given organism have the same DNA sequence that codes for the same genes, 
			different cell types of that organism only have a subset of genes “turned on”. 
			Genes are commonly annotated into pathways for summarising their collective effect in the biological systems. 
			One of the main drawbacks in current pathway annotation is that they are NOT cell type-specific.<br><br>
			
			We propose to identify and curate cell type-specific pathways, for example, in embryonic stem cells (ESCs) using the
			trans-omics datasets. The key assumption is that genes within a pathway should have correlated expression profile changes when perturbed.
			We have collected ESC differentiation data profiled in a time-course on both proteome and transcriptome levels. 
			Following the above assumption, we aim to (1) identify pathways that are regulated specifically in ESC differentiation; 
			and (2) curate ESC-specific pathways using statistical learning. 
			This project will expose you to the development and application of cutting-edge statistical 
			learning methods to the state-of-the-art bio-molecular applications. It sits at the heart of the interdisciplinary research conducted in our group. 			
        </div>
    </div>
</div>
<hr>


</div>
