---
layout: default
---

<div class="container">
    <h2>Notations:</h2>
    <br>
    		&#10018;: Co-first author<br>
    		&dagger;: Corresponding/Co-corresponding author<br>
    
    <hr>

	<h2 class="publications">Key publications</h2>
	<br>
			<b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R.,
			Oldfield, A., Kumar, D., Perera, D., Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup> (2019)
			Multi-omic profiling reveals dynamics of the phased progression of pluripotency.
			<b><i>Cell Systems</i></b>, 8(5), 427-445.
			[<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30115-2#%20">Full Text</a>]
			[<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
			<br><br>
			<p class="tab">&bull; <a href="https://f1000.com/prime/735727099?bd=1" target="_blank"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a></p>
			<br>

			Lin, Y., Ghazanfar, S., Wang, K., Gagnon-Bartsch, J., Lo, K., Su, X., Han, Z., Ormerod, J., Speed, T., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets.
			<b><i>Proceedings of the National Academy of Sciences of the United States of America</i></b>, 116(20), 9775-9784.
			[<a href="https://www.pnas.org/content/early/2019/04/25/1820006116">Full Text</a>]
			[<a href="https://bioconductor.org/packages/release/bioc/html/scMerge.html">BioC R package</a>]
			<br><br>	
			
			Cinghu, S.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Kosak, J., Conway, A., Kumar, D., Oldfield, A., Adelman, K. & Jothi, R. (2017)
    		Intragenic enhancers attenuate host gene expression.
			<b><i>Molecular Cell</i></b>, 68(1), 104–117.
			[<a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30658-5">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Attenuator.pdf">PDF</a>]<br><br>
			<p class="tab">&bull; highlighted in <b><i>Nature Reviews Genetics</i></b>, doi:10.1038/nrg.2017.90, 2017. [<a href="http://dx.doi.org/10.1038/nrg.2017.90">fulltext</a>]</p>
			<p class="tab">&bull; also highlighted in <b><i>Nature Reviews Molecular Cell Biology</i></b>, doi:10.1038/nrm.2017.111, 2017. [<a href="http://dx.doi.org/10.1038/nrm.2017.111">fulltext</a>]</p>
    		<br>

			Oldfield, A.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Conway, A., Cinghu, S., Freudenberg, J., Yellaboina, S. & Jothi, R. (2014). 
			Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. 
			<b><i>Molecular Cell</i></b>, 55(5), 708-722.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25132174">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/NFY.pdf">PDF</a>]
			<br><br>    
			
			<!-- 2 -->
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Humphrey, S., James, D., Yang, J. & Jothi, R.<sup>&dagger;</sup> (2016). 
			Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data.
			<b><i>Bioinformatics</i></b>, 32(2), 252-259.
       		[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26395771">Pubmed</a>], 
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Bioinformatics2016positiveUnlabelLearning.pdf">PDF</a>],
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/PUEL.html">Predictions</a>]
       		<br><br>
	<hr>
		
	
    <h2 class="publications">All publications</h2>
    <!--<h4> You can find a full publication list at <a href=https://scholar.google.com.au/citations?user=s-cliFIAAAAJ&hl=en target="_blank">Google Scholar.</a></h4>-->
    <br>
		<h3>2019</h3>
			<font color="#8B0000"><b>Systems biology:</b></font><br>

			56.
			Kim, H., Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
			Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning.
			<b><i>Nucleic Acids Research</i></b>, gkz1179.
			[<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
			<br><br>

			55. 
			Geddes, T., Kim, T., Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
			<b><i>BMC Bioinformatics</i></b>, accepted.
			[<a href="https://www.biorxiv.org/content/biorxiv/early/2019/09/19/773903.full.pdf">Full Text</a>]
			<br><br>

			54.
			Su, Z.<sup>&#10018;</sup>, Burchfield, J.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Humphrey, S., Yang, G., Francis, D., Yasmin, S., Shin, S., Norris, D., 
			Kearney, A., Astore, M., Scavuzzo, J., Fisher-Wellman, K., Wang, Q., Parker, B., Neely, G., Vafaee, F., Chiu, J., Yeo, R., Hogg, P., Fazakerley, D., Nguyen, L., Kuyucak, S. & James, D. (2019)
			Global redox proteome and phosphoproteome analysis reveals redox switch in Akt.
			<b><i>Nature Communications</i></b>, 10, 5486.
			[<a href="https://www.nature.com/articles/s41467-019-13114-4">Full Text</a>]
			<br><br>

			53. 
			<b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R.,
			Oldfield, A., Kumar, D., Perera, D., Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup> (2019)
			Multi-omic profiling reveals dynamics of the phased progression of pluripotency.
			<b><i>Cell Systems</i></b>, 8(5), 427-445.
			[<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30115-2#%20">Full Text</a>]
			[<a href="https://www.biorxiv.org/content/early/2018/09/19/415430.full.pdf">Biorxiv version</a>]
			[<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
			<br><br>
			
			52.
			Oldfield, A., Henriques, T., Kumar, D., Burkholder, A., Cinghu, S., Paulet, D., Bennett, B., <b><u>Yang, P.</b></u>, Scruggs, B., Lavender, C., Rivals, E., Adelman, K. & Jothi, R. (2019)
			NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nulceosome-depleted region. 
			<b><i>Nature Communications</i></b>, 10, 3072.
			[<a href="https://www.nature.com/articles/s41467-019-10905-7">Full Text</a>]
			<br><br>
			
			51.
			Parker, B.<sup>&#10018;</sup>, Calkin, A.<sup>&#10018;</sup>, Seldin, M.<sup>&#10018;</sup>, Keating, M. Tarling, E., <b><u>Yang, P.</u></b>, Moody, S., Liu, Y., Zerenturk, E.,
			Needham, E., Miller, M., Clifford, B., Morand, P., Watt, M., Meex, R., Peng, K., Lee, R., Jayawardana, K., Pan, C.,
			Mellett, N., Weir, J., Lazarus, R., Lusis, A., Meikle, P., James, D., Vallim, T. & Drew, B. (2019)
			An integrative systems genetic analysis of mammalian lipid metabolism.
			<b><i>Nature</i></b>, 567, 187-193.
			[<a href="https://www.nature.com/articles/s41586-019-0984-y">Full Text</a>]
    		<br><br>
			
			50.
			Azimi, A., <b><u>Yang, P.</u></b>, Ali, M., Howard, V., Mann, G., Kaufman, K., & Fernandez-Penas, P. (2019)
			Data independent acquisition proteomic analysis can discriminate between actinic keratosis, Bowen’s disease and cutaneous squamous cell carcinoma.
			<b><i>Journal of Investigative Dermatology</i></b>, 140(1), 212–222.e11
			[<a href="https://doi.org/10.1016/j.jid.2019.06.128">Full Text</a>]
			<br><br>
			<p class="tab">&bull; <a href="https://doi.org/10.3410/f.736074090.793562184" target="_blank"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a></p>
    		<br>
    		
    		49.
    		O'Sullivan, J., Neylon, A., Fahy, E., <b><u>Yang, P.</u></b>, McGorrian, C. & Blake, G. (2019)
    		MiR-93-5p is a novel predictor of coronary in-stent restenosis.
    		<b><i>Heart Asia</i></b>, 11(1), e011134.
    		[<a href="http://dx.doi.org/10.1136/heartasia-2018-011134">Full Text</a>]
			<br><br>
			
    		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
    		
    		48. Lin, Y., Ghazanfar, S., Strbenac, D., Wang, A., Patrick, E., Lin, D., Speed, T., Yang, J.<sup>&dagger;</sup> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
    		Evaluating stably expressed genes in single cells.
    		<b><i>GigaScience</i></b>, 8(9), giz106.
    		[<a href="http://shiny.maths.usyd.edu.au/SEGs">Interactive tool for SEG download</a>]
    		[<a href="http://gigadb.org/dataset/100637">Giga DB</a>]
    		[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_human.xlsx">human HK genes</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_mouse.xlsx">mouse HK genes</a>]
    		<br><br>
    		
			47. 
			Lin, Y., Ghazanfar, S., Wang, K., Gagnon-Bartsch, J., Lo, K., Su, X., Han, Z., Ormerod, J., Speed, T., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets.
			<b><i>Proceedings of the National Academy of Sciences of the United States of America</i></b>, 116(20), 9775-9784.
			[<a href="https://www.pnas.org/content/early/2019/04/25/1820006116">Full Text</a>]
			[<a href="https://bioconductor.org/packages/release/bioc/html/scMerge.html">BioC R package</a>]
			<br><br>		

			46.
			Kim, T., Chen, I., Parker, B., Humphrey, S., Crossett, B., Cordwell, S., <b><u>Yang, P.</u></b><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			QCMAP: An interactive web-tool for performance diagnosis and prediction of LC-MS Systems.
			<b><i>Proteomics</i></b>, 19(3), 1900068.
			[<a href="https://doi.org/10.1002/pmic.201900068">Full Text</a>]
			[<a href="http://shiny.maths.usyd.edu.au/QCMAP/">QCMAP tool</a>]
			<br><br>
			
			45.
			<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Ormerod, J., Liu, W., Ma, C., Zomaya, A. & Yang, J. (2019) 
			AdaSampling for positive-unlabeled and label noise learning with bioinformatics applications.
			<b><i>IEEE Transactions on Cybernetics</b></i>, 49(5), 1932-1943.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/AdaSampling_v11.pdf">PDF</a>],
			[<a href="https://github.com/PengyiYang/AdaSampling">Repo</a>]
			<br><br>

			44.
			Ridder, M., Klein, K., Yang, J., <b><u>Yang, P.</u></b>, Lagopoulos, J., Hickie, I., Bennett, M. & Kim, J. (2019)
			An uncertainty visual analytics framework for fMRI functional connectivity.
			<b><i>Neuroinformatics</b></i>, 17(2), 211-223.
			[<a href="https://doi.org/10.1007/s12021-018-9395-8">PDF</a>]
			<br><br>

		<h3>2018</h3>
			<font color="#8B0000"><b>Systems biology:</b></font><br>
			43.
			Fazakerley, DJ., Chaudhuri, R., <b><u>Yang, P.</u></b>, Maghzal, GJ., Thomas, KC., Krycer, JR., Humphrey, SJ., Parker, BL., Fisher-Wellman, KH., 
			Meoli, CC., Hoffman, NJ., Diskin, C., Burchfield, JG., Cowley, MJ., Kaplan, W., Modrusan, Z., Kolumam, G., Yang, JYH, 
			Chen, DL., Samocha-Bonet, D., Greenfield, JR., Hoehn, KL., Stocker, R. & James, DE. (2018) 
			Mitochondrial CoQ deficiency is a common driver of mitochondrial oxidants and insulin resistance.	
			<b><i>eLIFE</i></b>, 7, e32111.
			[<a href="https://elifesciences.org/articles/32111">Full Text</a>]
    		<br><br>
    		
    		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
    		42.
			Kim, T., Chen, I., Lin, Y., Wang, A., Yang, J. & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2018)
			Impact of similarity metrics on single-cell RNA-seq data clustering.
			<b><i>Briefings in Bioinformatics</i></b>, doi: 10.1093/bib/bby076
			[<a href="https://www.researchgate.net/publication/326769441_Impact_of_similarity_metrics_on_single-cell_RNA-seq_data_clustering">draft</a>]
			[<a href="https://github.com/SydneyBioX/scdney">Repo</a>]
    		<br><br>
    		
    	<h3>2017</h3>
    		<font color="#8B0000"><b>Systems biology:</b></font><br>
    		<!--
			42.
			Lin, Y., Ghazanfar, S., Strbenac, D., Wang, A., Patrick, E., Speed, T., Yang J.<sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2017)
    		Housekeeping genes, revisited at the single-cell level.
			doi: https://doi.org/10.1101/229815
			[<a href="https://www.biorxiv.org/content/early/2017/12/06/229815.full.pdf+html">PDF</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_human.xlsx">human HK genes</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_mouse.xlsx">mouse HK genes</a>]
    		<br><br>-->
    		
    		41.
    		Cinghu, S.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Kosak, J., Conway, A., Kumar, D., Oldfield, A., Adelman, K. & Jothi, R. (2017)
    		Intragenic enhancers attenuate host gene expression.
			<b><i>Molecular Cell</i></b>, 68(1), 104–117.
			[<a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30658-5">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Attenuator.pdf">PDF</a>]
    		<br><br>
    		
    		40.
    		Norris, DM.<sup>&#10018;</sup>, <b><u>Yang, P.</u></b><sup>&#10018;</sup>, Krycer JR., Fazakerley DJ, James, DE. & Burchfield JG.<sup>&dagger;</sup> (2017)
    		An improved Akt reporter reveals intra- and inter- cellular heterogeneity and oscillations in signal transduction.
			<b><i>Journal of Cell Science</i></b>, 130, 2757-2766.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/2757.full.pdf">PDF</a>]
    		<br><br>
    		
    		39.
    		<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Oldfield, A., Kim, T., Yang, A., Yang, J. & Ho, J.<sup>&dagger;</sup> (2017)
    		Integrative analysis identifies co-dependent gene expression regulation of BRG1 and CHD7 at distal regulatory sites in embryonic stem cells.
			<b><i>Bioinformatics</i></b>, 33(13), 1916–1920.
			[<a href="https://academic.oup.com/bioinformatics/article/2995820/Integrative-analysis-identifies-co-dependent-gene">Full Text</a>], 
			[<a href="http://pad2.maths.usyd.edu.au/PADv1/">Interactive PAD</a>]
    		<br><br>
    		
    		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			38.
			<b><u>Yang, P.</u></b>, Liu, W. & Yang, J. (2017) 
			Positive unlabeled learning via wrapper-based adaptive sampling.
			<i>Proceedings of the 26th International Joint Conference on Artificial Intelligence <b>(IJCAI)</b></i>, 
			Pre-print [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/IJCAI17_Yang.pdf">PDF</a>]
			<br><br>

    
		<h3>2016</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			37.
			Zheng, X., <b><u>Yang, P.</u></b>, Lackford, B., Bennett, B., Wang, L., Li, H., Wang, Y., Miao, Y., Foley, J., Fargo, D., 
			Jin, Y., Williams, C., Jothi, R. & Hu, G. (2016)
			CNOT3-dependent mRNA deadenylation safeguards the pluripotent state.
			<b><i>Stem Cell Reports</i></b>, 7(5), 897-910.
			[<a href="http://www.cell.com/stem-cell-reports/abstract/S2213-6711(16)30213-2">Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Zheng and Yang Stem Cell Reports 2016.pdf">PDF</a>]
			<br><br>
			
			36.
			Minard, A., Tan, S., <b><u>Yang, P.</b></u>, Fazakerley, D., Domanova, W., Parker, B., Humphrey, S., Jothi, R., Stöckli. J. & James. D. (2016)
			mTORC1 is a major regulatory node in the FGF21 signaling network in adipocytes.
			<b><i>Cell Reports</i></b>, 17(1), 29-36.
			[<a href="https://www.ncbi.nlm.nih.gov/pubmed/27681418">Pubmed</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			35.
			<b><u>Yang, P.</b></u>, Patrick, E., Humphrey, S., Ghazanfar, S., James, D., Jothi, R. & Yang, J. (2016). 
			KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis.
			<b><i>Proteomics</i></b>, 16(13), 1868-1871.
			[<a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201600068/full">Text</a>], 
			[<a href="http://kinasepa.pengyiyang.org">Online tool</a>]
			<br><br>
			
			34.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Humphrey, S., James, D., Yang, J. & Jothi, R.<sup>&dagger;</sup> (2016). 
			Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data.
			<b><i>Bioinformatics</i></b>, 32(2), 252-259.
       		[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26395771">Pubmed</a>], 
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Bioinformatics2016positiveUnlabelLearning.pdf">PDF</a>],
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/PUEL.html">Predictions</a>]
       		<br><br>
		
			33.
			Lu, C., Wang, J., Zhang, Z., <b><u>Yang, P.</b></u> & Yu, G. (2016).
			NoisyGOA: Noisy GO annotations prediction using taxonomic and semantic similarity.
			<b><i>Computational Biology and Chemistry</i></b>, 65, 203-211.
			<br><br>
					
			32.
			Domanova, W., Krycer, J., Chaudhuri, R., <b><u>Yang, P.</b></u>, Vafaee, F., Fazakerley, D., Humphrey, S., James, D. 
			& Kuncic, Z., (2016). Unraveling kinase activation dynamics using kinase-substrate relationships from temporal large-scale phosphoproteomics studies.
			<b><i>PLoS One</i></b>, 11(6), e0157763.

			<!--
			<font color="#FF0000">Practical contribution:</font><font color="#000000"> 
			this study provides a graphical user interface (GUI)-based syste for predicting substrates of kinases by integrating dynamic phosphoproteomics data with kinase recognition motif. <br>		
		
			<font color="#FF0000">Code available from:</font><font color="#000000">
	    	</font><a href="https://github.com/PengyiYang/KSP-PUEL">https://github.com/PengyiYang/KSP-PUEL</a><font color="#000000"><br>
			-->

		<h3>2015</h3>	

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			31.
			Pathania, R., Ramachandran, S., Elangovan, S., Padia, R., <b><u>Yang, P.</b></u>, Cinghu, S., Veeranan-Karmegam, R., Fulzele, S., 
			Pei, L., Chang, C., Choi, J., Shi, H., Manicassamy, S., Prasad, P., Sharma, S., Ganapathy, V., Jothi, R. & Thangaraju, M. (2015). 
			DNMT1 is essential for mammary and cancer stem cell maintenance and tumorigenesis. 
			<b><i>Nature Communications</i></b>, 6, 6910.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25908435">Pubmed</a>]
			<br><br>
			
			30.
			Hoffman, N., Parker, B., Chaudhuri, R., Fisher-Wellman, K., Kleinert, M., Humphrey, S., <b><u>Yang, P.</b></u>, Holliday, M., Trefely, S., 
			Fazakerley, D., Stockli, J., Burchfield, J., Jensen, T., Jothi, R., Kiens, B., Wojtaszewski, J., Richter, E. & James, D. (2015). 
			Global phosphoproteomic analysis of human skeletal muscle reveals a network of exercise-regulated kinases and AMPK substrates. 
			<b><i>Cell Metabolism</i></b>, 22(5), 922-935.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26437602">Pubmed</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			29.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). 
			Knowledge-based analysis for detecting key signaling events from time-series phosphoproteomics data. 
			<b><i>PLoS Computational Biology</i></b>, 11(8), e1004403.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26252020">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/knowledge-based kinase detection.pdf">PDF</a>]

		<h3>2014</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			28.
			Oldfield, A.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Conway, A., Cinghu, S., Freudenberg, J., Yellaboina, S. & Jothi, R. (2014). 
			Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. 
			<b><i>Molecular Cell</i></b>, 55(5), 708-722.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25132174">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/NFY.pdf">PDF</a>]
			<br><br>
			
			27.
			Ma, X., <b><u>Yang, P.</b></u>, Kaplan, W., Lee, B., Wu, L., Yang, J., Yasunaga, M., Sato, K., Chisholm, D. & James, D. (2014). 
			ISL1 regulates peroxisome proliferator-activated receptor &gamma; activation and early adipogenesis via bone morphogenetic protein 
			4-dependent and -independent mechanisms.
			<b><i>Molecular and Cellular Biology</i></b>, 34(19), 3607-3617.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25047837">Pubmed</a>]
			<br><br>
			
			26.
			Lackford, B., Yao, C., Charles, G., Weng, L., Zheng, X., Choi, E., Xie, X., Wan, J., Xing, Y., Freudenberg, J., 
			<b><u>Yang, P.</b></u>, Jothi, R., Hu, G. & Shi, Y. (2014). 
			Fip1 regulates mRNA alternative polyadenylation to promote stem cell self‐renewal. 
			<b><i>EMBO Journal</i></b>, 33(8), 878-889.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24596251">Pubmed</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			25.
			<b><u>Yang, P.</b></u><sup>&#10018;</sup>, Patrick, E.<sup>&#10018;</sup>, Tan, S., Fazakerley, D., Burchfield, J., Gribben, C., Prior, M., James, D. & Yang, J. (2014). 
			Direction pathway analysis of large-scale proteomics data reveals novel features of the insulin action pathway. 
			<b><i>Bioinformatics</i></b>, 30(6), 808-814.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24167158">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Direction.pdf">PDF</a>]
			<br><br>
			
			24.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Yoo, P., Fernando, J., Zhou, B., Zhang, Z. & Zomaya, A. (2014). 
			Sample subset optimization techniques for imbalanced and ensemble learning problems in bioinformatics applications. 
			<b><i>IEEE Transactions on Cybernetics</i></b>, 44(3), 445-455.
			[<a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6615954&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6615954">IEEE Xplore</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/SSO for imbalanced and ensemble learning.pdf">PDF</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/Sampling.html">Software</a>]

		<h3>2013</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			23.
			Humphrey, S., Yang, G., <b><u>Yang, P.</b></u>, Fazakerley, D. J., Stöckli, J., Yang, J. & James, D. (2013). 
			Dynamic adipocyte phosphoproteome reveals that Akt directly regulates mTORC2. 
			<b><i>Cell Metabolism</i></b>, 17(6), 1009-1020.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/23684622">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Adipocyte phosphoproteome.pdf">PDF</a>]
			<br><br>

			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			22.
			<b><u>Yang, P.</b></u>, Liu, W., Zhou, B., Chawla, S. & Zomaya, A. (2013). 
			Ensemble-based wrapper methods for feature selection and class imbalance learning. 
			In <i>Proceedings of the 17th Pacific-Asia Conference on Knowledge Discovery and Data Mining <b>(PAKDD)</b></i>, 
			Lecture Notes in Artificial Intelligence 7818, Springer Berlin Heidelberg, 544-555.
			[<a href="http://link.springer.com/chapter/10.1007%2F978-3-642-37453-1_45">Text</a>]
			<br><br>
			
			21.
			<b><u>Yang, P.</b></u>, Yang, J., Zhou, B. & Zomaya. A. (2013).
			Stability of feature selection algorithms and ensemble feature selection methods in bioinformatics. 
			In <i>Biological Knowledge Discovery Handbook: Preprocessing, Mining and Postprocessing of Biological Data</i>, 
			Wiley, New Jersey, USA, 333-352.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Yang_et_al_stability.pdf">PDF</a>]

			
		<h3>2012</h3>

			20.
			<b><u>Yang, P.</b></u>, Humphrey, S., Fazakerley, D., Prior, M., Yang, G., James, D. & Yang, J. (2012). 
			Re-fraction: a machine learning approach for deterministic identification of protein homologues and splice variants in 
			large-scale MS-based proteomics. 
			<b><i>Journal of Proteome Research</i></b>, 11(5), 3035-3045.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22428558">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/pr300072j.pdf">PDF</a>]
			<br><br>
			
			19.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Ma, J., Wang, P., Zhu, Y., Zhou, B. & Yang, J. (2012). 
			Improving X! Tandem on peptide identification from mass spectrometry by self-boosted Percolator. 
			<b><i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i></b>, 9(5), 1273-1280.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22689082">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/self-boosted percolator for xtandem.pdf">PDF</a>]
			<br><br>	
			
			18.
			Wang, P., <b><u>Yang, P.</b></u> & Yang, J. (2012). 
			OCAP: an open comprehensive analysis pipeline for iTRAQ. 
			<b><i>Bioinformatics</i></b>, 28(10), 1404-1405.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22467906">Pubmed</a>]

			
		<h3>2011</h3>

			17.
			<b><u>Yang, P.</b></u><sup>&#10018;, &dagger;</sup>, Ho, J.<sup>&#10018;</sup>, Yang, J. & Zhou, B. (2011). 
			Gene-gene interaction filtering with ensemble of filters. 
			<b><i>BMC Bioinformatics</i></b>, 12, S10.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21342539">Pubmed</a>]
			<br><br>
			
			16.
			<b><u>Yang, P.</b></u>, Zhang, Z., Zhou, B. & Zomaya, A. (2011). 
			Sample subset optimization for classifying imbalanced biological data. 
			In <i>Proceedings of the 15th Pacific-Asia Conference on Knowledge Discovery and Data Mining <b>(PAKDD)</b></i>, 
			Lecture Notes in Artificial Intelligence 6635, Springer Berlin Heidelberg, 333-344.
			[<a href="http://link.springer.com/chapter/10.1007%2F978-3-642-20847-8_28">Text</a>]


		<h3>2010</h3>

			15.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Ho, J., Zomaya, A. & Zhou, B. (2010). 
			A genetic ensemble approach for gene-gene interaction identification. 
			<b><i>BMC Bioinformatics</i></b>, 11(1), 524.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/20961462">Pubmed</a>]
			<br><br>
			
			14.
			Wang, P., <b><u>Yang, P.</b></u>, Arthur, J. & Yang, J. (2010). 
			A dynamic wavelet-based algorithm for pre-processing tandem mass spectrometry data. 
			<b><i>Bioinformatics</i></b>, 26(18), 2242-2249.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/20628072">Pubmed</a>]
			<br><br>
			
			13.
			Yoo, P., Ho, Y., Ng, J., Charleston, M., Saksena, N., <b><u>Yang, P.</b></u> & Zomaya, A. (2010). 
			Hierarchical kernel mixture models for the prediction of AIDS disease progression using HIV structural gp120 profiles. 
			<b><i>BMC Genomics</i></b>, 11, S22.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/21143806">Pubmed</a>]
			<br><br>
			
			12.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zhang, Z., Zhou, B. & Zomaya, A. (2010). 
			A clustering based hybrid system for biomarker selection and sample classification of mass spectrometry data. 
			<b><i>Neurocomputing</i></b>, 73(13), 2317-2331.
			[<a href="www.sciencedirect.com/science/article/pii/S0925231210002407">Text</a>]
			<br><br>
			
			11.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zhou, B., Zhang, Z. & Zomaya, A. (2010). 
			A multi-filter enhanced genetic ensemble system for gene selection and sample classification of microarray data. 
			<b><i>BMC Bioinformatics</i></b>, 11, S5.
			[<a href="www.sciencedirect.com/science/article/pii/20122224">Pubmed</a>]
			<br><br>
			
			10.
			<b><u>Yang, P.</b></u>, Yang, J., Zhou, B. & Zomaya, A. (2010). 
			A review of ensemble methods in bioinformatics. 
			<b><i>Current Bioinformatics</i></b>, 5(4), 296-308.
			[<a href="http://benthamscience.com/journals/current-bioinformatics/volume/5/issue/4/page/296/">Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/EnsembleBioinformatics-v6.pdf">PDF</a>]
			<br><br>			
			
			9.
			Li, L., <b><u>Yang, P.</b></u>, Ou, L., Zhang, Z. & Cheng, P. (2010). 
			Genetic algorithm-based multi-objective optimisation for QoS-aware web services composition. 
			In <i>Proceedings of the 4th International Conference on Knowledge Science, Engineering and Management <b>(KSEM)</b></i>, 
			Lecture Notes in Computer Science 6291, Springer Berlin Heidelberg, 549-554.
			[<a href="link.springer.com/chapter/10.1007%2F978-3-642-15280-1_50">Text</a>]

			
		<h3>2009</h3>

			8.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Xu, L., Zhou, B., Zhang, Z. & Zomaya, A. (2009). 
			A particle swarm based hybrid system for imbalanced medical data sampling. 
			<b><i>BMC Genomics</i></b>, 10, S34.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/19958499">Pubmed</a>]
			<br><br>
			
			7.
			Zhang, Z., <b><u>Yang, P.</b></u>, Wu, X. & Zhang, C. (2009). 
			An agent-based hybrid system for microarray data analysis. 
			<b><i>IEEE Intelligent Systems</i></b>, 24(5), 53-63.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30029412/zhang-agentbasedhybridsystem-2009.pdf">PDF</a>]
			<br><br>
			
			6.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup> & Zhang, Z. (2009). An embedded two-layer feature selection approach for microarray data analysis. 
			<b><i>IEEE Intelligent Informatics Bulletin</i></b>, 10(1), 24-32.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30029413/zhang-embeddedtwolayer-2009.pdf">PDF</a>]
			<br><br>
			
			5.
			<b><u>Yang, P.</b></u>, Tao, L., Xu, L. & Zhang, Z. (2009). 
			Multiagent framework for bio-data mining. 
			In <i>Proceedings of the 4th Rough Sets and Knowledge Technology <b>(RSKT)</b></i>, 
			Lecture Notes in Computer Science 5589, Springer Berlin Heidelberg, 200-207.
			[<a href="http://link.springer.com/chapter/10.1007%2F978-3-642-02962-2_25">Text</a>]
			

			
		<h3>2008</h3>

			4.
			Zhang, Z. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2008). 
			An ensemble of classifiers with genetic algorithm-based feature selection. 
			<b><i>IEEE Intelligent Informatics Bulletin</i></b>, 9(1), 18-24.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30017964/zhang-anemembleofclassifiers-2008.pdf">PDF</a>]
			<br><br>
			
			3.
			<b><u>Yang, P.</b></u> & Zhang, Z. (2008). 
			A clustering based hybrid system for mass spectrometry data analysis. 
			In <i>Proceedings of the 3rd Pattern Recognition in Bioinformatics <b>(PRIB)</b></i>, 
			Lecture Notes in Bioinformatics 5265, Springer Berlin Heidelberg, 98-109.
			[<a href="http://link.springer.com/chapter/10.1007%2F978-3-540-88436-1_9">Text</a>]
			<br><br>
			
			
			2. 
			<b><u>Yang, P.</b></u> & Zhang, Z. (2008).
			A hybrid approach to selecting susceptible single nucleotide polymorphisms for complex disease analysis. 
			In <i>Proceedings of BioMedical Engineering and Informatics <b>(BMEI)</b></i>, IEEE, 214-218.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30018221/zhang-ahybridapproach-2008.pdf	">PDF</a>]		

			
		<h3>2007</h3>
			1. <b><u>Yang, P.</b></u> & Zhang, Z. (2007). 
			Hybrid methods to select informative gene sets in microarray data classification. 
			In <i>Proceedings of the 20th Australian Joint Conference on Artificial Intelligence <b>(AI)</b></i>, 
			Lecture Notes in Artificial Intelligence 4830, Springer Berlin Heidelberg, 811-815.
			[<a href="http://link.springer.com/chapter/10.1007%2F978-3-540-76928-6_97">Text</a>]	

</div>
